### PR TITLE
-ScanAvgCPULoadFactor

### PR DESCRIPTION
**Pull request by andrewjohnporter**
Is it a hard limit or not. this document doesn't say if it is or it isn't but this doc say it's not a hard limit so have edited this one. Need someone to confirm which is true: https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-antivirus/configure-advanced-scan-types-microsoft-defender-antivirus

### CODE REVIEW

**Review by get-itips — APPROVED** [review]
The PR that added that sentence in the linked article is here https://github.com/MicrosoftDocs/windows-itpro-docs/pull/376 and was approved by several Microsoft people, will accept this one.

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by e0i** [comment]
cc @JohanFreelancer9 Final copy editing may be needed before merging this PR. Thank you.

File: docset/windows/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -752,6 +752,8 @@ The acceptable values for this parameter are: integers from 5 through 100, and t
 Windows Defender does not exceed the percentage of CPU usage that you specify.
 The default value is 50.
 
+Note: This is not a hard limit but rather a guidance for the scanning engine to not exceed this maximum on average.
```
Comment (JohanFreelancer9) [inline docset/windows/defender/Set-MpPreference.md:755]:
```suggestion
> [!NOTE]
> This is not a hard limit but rather a guidance for the scanning engine to not exceed this maximum on average.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: docset/windows/defender/Set-MpPreference.md (hunk 1 above)
Comment (tseward) [reply docset/windows/defender/Set-MpPreference.md:755]: Are we including markdown, now?

Comment (JohanFreelancer9) [reply docset/windows/defender/Set-MpPreference.md:755]: Hi @tseward - my suggestion was purely for the sake of consistency to align the format of the note with the format of the note further down in this topic. However, I am happy to leave it as is.

**Pull request merged**